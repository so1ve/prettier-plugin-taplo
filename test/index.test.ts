import { format } from "prettier";
import { describe, expect, it } from "vitest";

const testFormat = async (code: string) =>
	await format(code, {
		plugins: ["./dist/index.cjs"],
		parser: "taplo",
	});

describe("should", () => {
	it("exported", async () => {
		await expect(
			testFormat(`
[foo]  
asfd=            1
`),
		).resolves.toMatchSnapshot();
	});
});
