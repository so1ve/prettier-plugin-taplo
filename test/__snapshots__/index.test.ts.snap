// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should > exported 1`] = `
"[foo]
asfd = 1
"
`;
